help/_layouts/default.html

```diff
@@ -5,247 +5,67 @@
-            font-size: 20px;
-            margin-bottom: 12px;
-        }
-
-        main p {
-            font-size: 16px;
-            line-height: 1.6;
-            margin-bottom: 20px;
-        }
-
-        .is-active-link {
-            font-weight: normal;
-        }
-
-        .scroll-spacer {
-            height: 300px;
-        }
-
-        /* Footer */
-        footer {
-            margin-left: 300px;
-            color: #0366d6;
-            background-color: #f9fafb;
-            padding: 40px 20px;
-            font-size: 14px;
-            max-width: 900px;
-        }
-
-        footer h3 {
-            color: #0366d6;
-        }
-
-        footer ul {
-            list-style: none;
-            padding: 0;
-        }
-
-        footer ul li a {
-            color: #0366d6;
-            text-decoration: none;
-        }
-
-        footer ul li a:hover {
-            text-decoration: underline;
-        }
-
-        footer .social-icons a img {
-            width: 20px;
-            margin-right: 10px;
-        }
-
-        .footer-container {
-            display: flex;
-            justify-content: center;
-            max-width: 1200px;
-            margin: 0 auto;
-        }
-
-        .footer-column {
-            flex: 1;
-            max-width: 300px; /* Set a max-width for each column */
-            padding: 0 20px;  /* Add padding for some space between the columns */
-        }
-    </style>
+    <link rel="stylesheet" href="default.css">
 </head>
 <body>
 
     <!-- Header -->
     <header class="header">
         <div class="logo">
+            <div class="hamburger" id="hamburger">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+
             <a href="/">Expensify</a>
+            /
+            <!-- Dropdown for product selection -->
+            <div class="dropdown">
+                <span>{{ page.title | remove_first: 'Expensify ' }}</span>
+                <div class="dropdown-content">
+                    <a href="/expense" class="{% if page.title == 'Expensify Expense' %}active{% endif %}">Expense</a>
+                    <a href="/card" class="{% if page.title == 'Expensify Cards' %}active{% endif %}">Card</a>
+                    <a href="/travel" class="{% if page.title == 'Expensify Travel' %}active{% endif %}">Travel</a>
+                    <a href="/chat" class="{% if page.title == 'Expensify Chat' %}active{% endif %}">Chat</a>
+                    <a href="/invoice" class="{% if page.title == 'Expensify Invoice' %}active{% endif %}">Invoice</a>
+                    <a href="/billpay" class="{% if page.title == 'Expensify Billpay' %}active{% endif %}">Billpay</a>
+                </div>
+            </div>
+        </div>
+        <div id="search-container">
+            <!-- Search Input styled as button -->
+            <span id="search-icon" style="cursor: pointer; font-size: 24px;">🔍</span>
```

Comment:
Please place all styles in the CSS file and apply them with classes or IDs as necessary.